jenkins_pipelines/environments/common/pipeline-sonarqube.groovy

```diff
@@ -0,0 +1,30 @@
+def run(params) {
+    stage('Clone project') {
+        git url: params.git_repo, branch: params.git_branch
+        dir ("uyuni") {
+            checkout scm
+        }
```

Comment:
You don't need this. This will checkout susemanager-ci repository, as far as I understand you don't need it.
It's just fine with your previous line.

Reply:
OK